### DISCUSSION TITLE
can't install from the AUR

### DISCUSSION THREAD

**Discussion by toxaxy** [vit1251/golden#175] [General]
can't install from the AUR.
==> ERROR: Failure while downloading https://github.com/vit1251/golden-point/archive/refs/tags/1.2.19.tar.gz

**Comment by vit1251** [comment]
Defect register as issue https://github.com/vit1251/golden/issues/176.
Source code repository address is https://github.com/vit1251/golden/archive/refs/tags/1.2.19.tar.gz
You may change manually AUR source code or watch referred issue.
